README.md

```diff
@@ -0,0 +1,9 @@
+Welcome to our new repository! As long anticipated, we have finally merged all three repositories (Cumulus, Polkadot and Substrate) into a single unified entity, marking a significant milestone in our journey.
```

Comment:
IMO, we should write a proper `README`. This means no welcoming and that this is a new repo etc.

Reply:
okay.